theano/gpuarray/dnn.py

```diff
@@ -2833,6 +2833,295 @@ def local_abstractconv3d_cudnn_graph(op, context_name, inputs, outputs):
     return [rval]
 
 
+class GpuDnnTransformerDescriptor(COp):
+
+    """
+    This Op builds a spatial transformer descriptor for use in spatial transformer network
+    operations.
+    """
+
+    __props__ = ('dtype',)
```

Comment:
I've removed dtype from all of them, except the descriptor, following the design of ``GpuDnnConv``, as the cuDNN datatype (``cudnnDataType_t``) is required in the descriptor setup.